### DISCUSSION TITLE
Does it take in Raw h264 byte stream/packets?

### DISCUSSION THREAD

**Discussion by yuanxintaxon** [bluenviron/mediamtx#827] [Questions]
Hi all. May I ask does the RTSP server takes in raw h264 data in NAL unit? (I’m currently building a C# app that will be taking in raw h264 byte stream from a drone camera).
My idea is to stream the live visualization from the drone camera to the RTSP server.

**Comment by aler9** [comment]
Hello, you have to wrap raw H264 packets inside RTP packets and perform an RTSP handshake. This can be done by using ffmpeg or Gstreamer or gortsplib.

[a comment by an automated account is not shown]
